doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1177,7 +1168,7 @@ a default ``__eq__`` and ``__hash__``. If you have a parametrized dtype, you sho
 update the ``ExtensionDtype._metadata`` tuple to match the signature of your
 ``__init__`` method. See :class:`pandas.api.extensions.ExtensionDtype` for more (:issue:`22476`).
```

Comment:
At the end of #22476, @TomAugspurger  and I were talking about an ambiguous sentence in the EA whatsnew: "Pandas now requires that extension dtypes be hashable."

How about: Pandas now requires that extension dtypes be hashable *(i.e. the respective `ExtensionDtype`-objects; hashability is not a requirement for the values of the corresponding `ExtensionArray`)*.